library/src/main/res/layout/chucker_fragment_transaction_payload.xml

```diff
@@ -68,4 +68,36 @@
         app:constraint_referenced_ids="emptyPayloadImage,emptyPayloadTextView"
         tools:visibility="visible" />
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab_to_next"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:backgroundTint="@color/chucker_fab_background_colour"
+        android:contentDescription="@string/chucker_scroll_buttons_for_search"
+        android:src="@drawable/chucker_ic_arrow_down"
+        android:visibility="gone"
+        app:fabSize="mini"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:tint="@color/chucker_color_on_primary"
+        tools:visibility="visible" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab_to_previous"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="4dp"
+        android:backgroundTint="@color/chucker_fab_background_colour"
+        android:contentDescription="@string/chucker_scroll_buttons_for_search"
+        android:enabled="false"
+        android:rotation="180"
+        android:src="@drawable/chucker_ic_arrow_down"
+        android:visibility="gone"
+        app:fabSize="mini"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/fab_to_next"
+        app:tint="@color/chucker_color_on_primary"
+        tools:enabled="false"
+        tools:visibility="visible" />
+
```

Comment:
> We could chose to display the search results info summary by extending the App Bar to also include the icons(chevrons) which could be used to traverse through the filtered search results.

This would be preferred IMHO. Specifically having the # of occurrences would be helpful (see my video in top comment also).

Also I would refrain from using `FloatingActionButton` as they're specific to MDC and I'm looking into having MDC being used as a `compileOnly` dependency. So using classes directly inside the XML file makes things a bit harder.

Reply:
Yeah, I agree.

Will do the changes to summarise the search results in the app bar, along with the navigation chevrons.